ios/Plugin/Plugin.swift

```diff
@@ -35,15 +33,73 @@ public class NativeAudio: CAPPlugin {
         if let fade = call.getBool(Constant.FadeKey) {
             self.fadeMusic = fade
         }
+
         if let focus = call.getBool(Constant.FocusAudio) {
+
             do {
+
                 if focus {
+
                     try self.session.setCategory(AVAudioSession.Category.playback)
-                } else {
-                    try self.session.setCategory(AVAudioSession.Category.ambient)
+
                 }
+
             } catch {
+
                 print("Failed to set setCategory audio")
+
+            }
+
+        }
+
+        if let background = call.getBool(Constant.Background) {
+
+            do {
+
+                if background {
```

Comment:
I’m not sure why the if is repeated here with line 56.

i think you can give a default value to the getter and then remove the if assign and the double scope